.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,24 @@
+name: Close Stale Issues
+
+on:
+  workflow_dispatch:
```

Comment:
Manually triggered to prevent GH APIs from being throttled.
The default is 30 operations at a time.